src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -22,21 +26,23 @@
   private HttpServletRequest mockRequest;
   private HttpServletResponse mockResponse;
   private RequestDispatcher mockRequestDispatcher;
-
+  
   @Before
-  public void setup() throws IOException {
+  public void setup() throws IOException, ServletException {
```

Comment:
Why did you need to add this?

Reply:
It was showing up as an error on eclipse